docs/UserGuide.md

```diff
@@ -40,95 +50,163 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task: `add`
+Adds a task to GGist<br>
+General format: `add TASK, [DATE], [TIME], [PRIORITY], [FREQUENCY]`
 
-> Persons can have any number of tags (including 0)
+<img src="images/2ndimpt.png" width="500"><br>
+
+###### With deadline
+Format: `add TASK, DATE, TIME, [PRIORITY], [FREQUENCY]`
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add write diary, jul 10, 1300`
+* `add prepare presentation slides, mon, 1400, high`
+
+###### Without any deadline
+Format: `add TASK, [DATE], [TIME], [PRIORITY], [FREQUENCY]`
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+Examples: 
+* `add buy milk, low`
+* `add buy present for brother's birthday`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+###### With start and end time
+Format: `add TASK, DATE, TIME, [PRIORITY], [FREQUENCY]`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+For this kind of task, the parameter TIME is in the format START-END.
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `add dad's birthday celebration, jul 10, 1900-2100, high`
+* `add company's D&D, sun, 1900-2200`
+
+###### Recurring
+To make tasks repeating, simply  add the FREQUENCY parameter at the back.
 
+Format: `add TASK, [DATE], [TIME], [PRIORITY], [FREQUENCY]`
```

Comment:
in fact, instead of repeating the format, you can simply leave one command format at the beginning and then explain each variation with example highlighting the parameter required.